Secret-agency-persistence/src/test/java/cz/fi/muni/pa165/secretagency/dao/DepartmentDaoTests.java

```diff
@@ -98,6 +104,22 @@ public void getById() {
         assertEquals(found.getLongitude(), 1d);
         assertEquals(found.getSpecialization(), DepartmentSpecialization.ASSASSINATION);
         assertEquals(found.getAgents().size(), 2);
+
+        Agent foundAgent1 = found.getAgents().get(0);
+        assertEquals(foundAgent1.getName(), a1.getName());
```

Comment:
Nestačí equals nad entitami?

Reply:
zkousel jsem a neslo mi to 😄 mozna je napsane blbe equals || hashCode, ale nevim 🤔

Reply:
Odstranil bych z něj missions a department.

Reply:
jo to funguje, asi to vlastne dava i smysl, ze nechceme porovnavat na zaklade department & mission